.github/workflows/check-partner-approved-label.yml

```diff
@@ -9,7 +9,7 @@ permissions:
 jobs:
   check_label_job:
     runs-on: ubuntu-latest
-    if: github.repository == 'demisto/content' && github.event.action == 'opened' && github.event.pull_request.head.repo.fork == true && contains(github.head_ref, 'xsoar-bot-contrib-ContributionTestPack') == false
+    if: github.repository == 'demisto/content' && github.event.pull_request.head.repo.fork == true && contains(github.head_ref, 'xsoar-bot-contrib-ContributionTestPack') == false
```

Comment:
So we want it to run not only when the PR is opened right? that is the fix.

Reply:
Yes, in this case the flow was running only when PR opened but not after the change of the branch